backend/src/Squidex/Program.cs

```diff
@@ -56,6 +56,10 @@ public static void Main(string[] args)
 
                         serverOptions.ListenAnyIP(5000);
                     }
+                    else
```

Comment:
Likely this will want to be expanded upon, just including in the initial changes to demonstrate how the application _needs_ to run on a non-privileged port (1-1024) as this requires root access.